### PR TITLE
Make write 30 times faster

### PR DESCRIPTION
**Pull request by maxfreu**
I just copied the ArchGDAL code for chunked writes here. Makes writing sqlite files 30 times faster for me :)

### CODE REVIEW

**Comment by evetion** [comment]
That's great stuff, thanks! FWIW, do we need these config options, as in, can we make it just the default?

**Comment by maxfreu** [comment]
We can make it the default, but if I recall correctly, there were issues with some vector drivers and some corner cases. That's why this flag exists. But of course we can set the default to false and just say set it to true if you run into troubles.

File: src/io.jl [hunk 1]
```diff
@@ -90,9 +90,19 @@ end
 """
     write(fn::AbstractString, table; layer_name="data", crs::Union{GFT.GeoFormat,Nothing}=crs(table), driver::Union{Nothing,AbstractString}=nothing, options::Vector{AbstractString}=[], geom_columns::Set{Symbol}=(:geometry))
 
-Write the provided `table` to `fn`. The `geom_column` is expected to hold ArchGDAL geometries.
+Write the provided `table` to `fn`. The `geom_column` is expected to hold ArchGDAL geometries. Experimental: Fast chunked writes can be enabled by setting `use_gdal_copy=true` and `chunksize` to the desired value (default 20000).
```
Comment (asinghvi17) [inline src/io.jl:93]:
```suggestion
Write the provided `table` to `fn`. The `geom_column` is expected to hold ArchGDAL geometries. 

Experimental: Fast chunked writes can be enabled by setting `use_gdal_copy=true` and `chunksize` to the desired value (default 20000).

**Comment by evetion** [comment]
I think I will adopt chunking to be default for the applicable drivers. You could disable that by setting the chunksize to zero? That makes more sense to me than several kwargs?

**Pull request closed without merging**